config.py

```diff
@@ -16,3 +15,39 @@
 # This dict holds aspect ratio data for hydrograph rendered in static/js/hydrograph.js. Keys are 'height' and 'width'
 HYDRO_META = {}
 
+# Map configuration
+
+SPATIAL_REFERENCE_ENDPOINT = 'http://spatialreference.org/ref/epsg/${epsg_code}/proj4/'
+PROJECTION_EPSG_CODE = '4326'
+
+#
+BOUNDING_BOX = [-94.294625, 41.015733, -91.424186, 43.102420]
+
+BACKGROUND_FILE = 'floodviz/static/geojson/states.json'
+REFERENCE_FILE = 'floodviz/static/geojson/reference.json'
```

Comment:
I would prefer for now, it just go directly into config.py. We need to discuss as a team how we really want to read in configuration parameters. For now, I'd like them collected in a single place.